src/en/mangago/src/eu/kanade/tachiyomi/extension/en/mangago/Mangago.kt

```diff
@@ -151,11 +151,17 @@ class Mangago : ParsedHttpSource() {
 
     override fun searchMangaNextPageSelector() = genreListingNextPageSelector
 
+    private val descriptionRegex = Regex("""The following content is intended for mature audiences and may contain sexual themes, gore, violence and/or strong language\. Discretion is advised\.""")
+    private fun cleanDescription(description: String) = description.replace(descriptionRegex, "").trim()
+
+    private val titleRegex = Regex("""\(yaoi\)|\{Official\}|«Official»|〘Official〙|\(Official\)|\s\[Official]|\s「Official」|『Official』|\s?/Official\b""")
+    private fun cleanTitle(title: String) = title.replace(titleRegex, "").substringBeforeLast("(").trim()
+
     override fun mangaDetailsParse(document: Document) = SManga.create().apply {
-        title = document.selectFirst(".w-title h1")!!.text()
+        title = cleanTitle(document.select(".w-title h1").text())
```

Comment:
make a preference for it

Reply:
tyvm for all of the examples! they helped a lot